src/platform/tls_schannel.c

```diff
@@ -1387,8 +1387,10 @@ QuicTlsUninitialize(
     )
 {
     if (TlsContext != NULL) {
-        QuicTraceLogVerbose("[ tls][%p][%c] Cleaning up.",
-            TlsContext, GetTlsIdentifier(TlsContext));
+        QuicTraceLogConnVerbose(

```

Comment:
How will we distinguish client events from server events?

Reply:
The connection pointer. It also allows us to programmatically collect all events for a given connection.

Reply:
Sounds good to me